generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java

```diff
@@ -164,6 +166,157 @@ public OpaqueExpressionGenerator(IRandom generator, GenerationParams generationP
         depth, fuzzer)));
   }
 
+  /**
+   * Function to generate an opaque zero or one by bitwise shifting left or right by an
```

Comment:
I'm not sure it's right to say that an arbitrary number of bits is used here.  In the code you take some care about the shift value (and I think maybe even more care needs to be taken; see comments below).

Reply:
Got it.